Android.mk

```diff
@@ -51,7 +51,7 @@ srcs +=	adbg/src/adbg_case.c \
 	regression_6000.c \
 	regression_8000.c \
 	regression_8100.c \
-	sha_perf.c \
+	digest_perf.c \
```

Comment:
I would prefer `hash_perf.c`. Please make one commit to rename the files and functions ("Rename --sha-perf to --hash-perf"). Then a second commit to introduce the new algorithms.

Reply:
@jforissier 
I am sorry that I don't understand your words clearly.If I don't change the interface, can I reuse this TA UUID for digest and mac?

Reply:
Yes.

Reply:
I have modified the code according to your comments and I will commit  after verification soon.